threads/serializers.py

```
@@ -40,17 +40,59 @@ def to_representation(self, instance):
+            attrs["images"] = attrs.pop("images",None)
+            if not attrs["images"]:
+                raise serializers.ValidationError({"images": "This field is required."})
+        elif type == CommentType.TEXT:
+            attrs["content"] = attrs.pop("content",None)
+            if not attrs["content"]:
+                raise serializers.ValidationError({"content": "This field is required."})
+        return super().validate(attrs)
+
+
+
+    def update(self, instance, validated_data):
```

Flake8 (7.2.0)

[error] too many blank lines (3)

(E303)